docs/run_publish/references.md

```diff
@@ -332,6 +334,18 @@ By default this is true. when set to false with:
 
 This removes the created_at and updated_at columns in the starter_entities table.
 
+### --unfinalized-blocks
+
+This will allow you to index blocks before they become finalized. It can be very useful if you want the most up-to-date data possible. It will detect any forks and remove any blocks that don't become finalized. By default it is set to `false`. To change it to `true` run following command: 
+
+```shell
+> subql-node -f . --unfinalized-blocks
+```
+
+::: info Note
+This feature is available for Substrate-based blochchains. 
```

Comment:
We also require historical indexing to be enabled

Reply:
Agree, add a link to the docs and note that you can determine this by checking if the projects logs the correct message at the startup sequence

Reply:
Updated.